README.md

```diff
@@ -3,33 +3,42 @@ websocket
 
 Packages websocket and websocketjs provide high- and low-level bindings for the browser's WebSocket API (respectively).
 
-The high-level bindings act like a regular `net.Conn`. They can be used as such. For example:
+The high-level bindings offer a Dial function that returns a regular net.Conn.
+It can be used similar to net package.
```

Comment:
I know everybody hates a grammar nazi, but this should read "It can be used **similarly** to net package", as it describes an action, so an adverb is called for.